application/dash_application/views/single_lane_rna.py

```diff
@@ -187,26 +188,24 @@ def get_rna_data():
      "value": PINERY_COL.SampleName}
 ]
 
-
+# TODO: Aww beans the subplots are gonna give me grief
```

Comment:
As all views will be switched over to subplots soon, the new ordering approach needs to work with subplots.

Reply:
that TODO was outdated, sorry for leaving it in :sweat_smile: